packages/calcite-components/src/components/date-picker/date-picker.e2e.ts

```diff
@@ -279,6 +280,44 @@ describe("calcite-date-picker", () => {
       expect(await monthOptions[3].getProperty("disabled")).toBe(false);
       expect(await monthOptions[4].getProperty("disabled")).toBe(true);
     });
+
+    it("disables days outside minAsDate and maxAsDate", async () => {
+      const page = await newE2EPage();
+      await page.emulateTimezone("America/Los_Angeles");
+      await page.setContent(html`<calcite-date-picker></calcite-date-picker>`);
+
+      const currentDay = new Date();
+
+      const maxDay = new Date();
+      maxDay.setDate(maxDay.getDate() + 2);
+
+      const outOfRangeDay = new Date();
+      outOfRangeDay.setDate(currentDay.getDate() + 5);
+
+      await page.evaluate(
+        (min, max) => {
+          const el = document.querySelector("calcite-date-picker");
+          el.minAsDate = new Date(min);
+          el.maxAsDate = new Date(max);
+        },
+        currentDay,
+        maxDay,
+      );
+      await page.waitForTimeout(2000);
+
+      expect(await (await getDayById(page, dateToISO(currentDay).replaceAll("-", ""))).getProperty("active")).toBe(
```

Comment:
Could you look at simplifying by setting an explicit start date and using stable day IDs (similar to [other usages of `getDayById`](https://github.com/Esri/calcite-design-system/blob/850fd0260b9cbbada28d48c7e916ef5993a32408/packages/calcite-components/src/components/date-picker/date-picker.e2e.ts#L564-L567))?